test/e2e/e2e_test.go

```diff
@@ -240,6 +248,231 @@ var _ = ginkgo.Describe("Kueue visibility server", func() {
 				}, util.Timeout, util.Interval).Should(gomega.BeComparableTo(wantPendingWorkloads, cmpopts.IgnoreFields(metav1.ObjectMeta{}, "Name")))
 			})
 		})
+
+		ginkgo.It("Should allow fetching information about pending workloads in LocalQueue", func() {
+			ginkgo.By("Verify there are zero pending workloads", func() {
+				gomega.Eventually(func() int {
+					info, err := visibilityClient.LocalQueues(nsA.Name).GetPendingWorkloadsSummary(ctx, localQueueA.Name, metav1.GetOptions{})
+					gomega.Expect(err).NotTo(gomega.HaveOccurred())
+					return len(info.Items)
+				}, util.Timeout, util.Interval).Should(gomega.Equal(0))
```

Comment:
Why do we need to use `Eventually` instead of `Expect`?

Reply:
That's a good point, fixed